src/go/rpk/pkg/cli/cloud/security/security.go

```diff
@@ -0,0 +1,24 @@
+package security
```

Comment:
Please hold the merge of this PR as we are discussing a potential new home for this command. I will update the PR if needed.